Formula/gazebo3.rb

```diff
@@ -3,7 +3,7 @@ class Gazebo3 < Formula
   homepage "http://gazebosim.org"
   url "https://osrf-distributions.s3.amazonaws.com/gazebo/releases/gazebo-3.2.0.tar.bz2"
   sha256 "6b77382a491833d5292b3e1fca34a04c968025a09746d87cdcf77cff040acea5"
-  head "https://bitbucket.org/osrf/gazebo", :branch => "default", :using => :hg
+  head "https://github.com/osrf/gazebo", :branch => "master"
```

Comment:
we don't have a `gazebo3` branch but `master` is not probably the code what a Gazebo archeologist would be looking for.

Reply:
thanks, I was being lazy but I fixed it in 6c43a6bba443c8a5967f548ed173fe17de55d8c8